.github/workflows/main.yml

```diff
@@ -76,3 +103,8 @@ jobs:
             run: dotnet run --project build/Build.csproj -- --target=DeployNuGetsToGithub
             env:
               GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+          - name: Push Marketplace Nugets
+            run: dotnet run --project build/Build.csproj -- --target=DeployNuGetsToMarketplace
+            env:
+              GITHUB_TOKEN: ${{ secrets.NUGET_API_KEY }}
```

Comment:
Make a deploy task in CAKE thats combined of multiple tasks like so:
- https://github.com/MonoGame/MonoGame/blob/develop/build/Tasks.cs#L4

Then add prerequirements for running both of those tasks on cake side like so:
- https://github.com/MonoGame/MonoGame/blob/develop/build/BuildFrameworksTasks/BuildAndroidTask.cs#L7